html/MANIFEST.in

```diff
@@ -0,0 +1 @@
+recursive-include avocado_result_html/resources *
```

Comment:
Btw this version does not work for me. I needed to change the paths to:

```
diff --git a/html/avocado_result_html/__init__.py b/html/avocado_result_html/__init__.py
index eaa4a4d..373277d 100644
--- a/html/avocado_result_html/__init__.py
+++ b/html/avocado_result_html/__init__.py
@@ -198,7 +198,7 @@ class HTMLResult(Result):
 
     @staticmethod
     def _copy_static_resources(html_path):
-        module = 'avocado.plugins.html.avocado_result_html'
+        module = 'avocado_result_html'
         base_path = 'resources/static'
 
         for top_dir in pkg_resources.resource_listdir(module, base_path):
@@ -233,7 +233,7 @@ class HTMLResult(Result):
     def _render(self, result, output_path):
         context = ReportModel(result=result, html_output=output_path)
         template = pkg_resources.resource_string(
-            'avocado.plugins.html.avocado_result_html',
+            'avocado_result_html',
             'resources/templates/report.mustache')
 
         # pylint: disable=E0611

```

then I run `make link` and then `cd html; python setup.py develop --user`. Then it seems to work.